tests/test-backend-ops.cpp

```diff
@@ -4840,7 +4840,7 @@ struct test_moe_expert_reduce : public test_case {
 
         std::vector<ggml_tensor *> expert_views(n_expert_used);
         for (int64_t i = 0; i < n_expert_used; ++i) {
-            expert_views[i] = ggml_view_2d(ctx, weighted, n_embd, n_tokens, weighted->nb[2], i * weighted->nb[1]);
+            expert_views[i] = ggml_view_2d(ctx, weighted, n_embd, n_tokens, weighted->nb[1], i * weighted->nb[1]);
```

Comment:
I don't think this change is correct.

Reply:
This update was testing some changes to the addition kernels in response to the discussion in https://github.com/ggml-org/llama.cpp/pull/16857. But, it looks like the CUDA CI is failing with this change too, so if it's confirmed that the `nb[2]` is correct here I'll need to do a little more debugging to understand why the WebGPU add op is failing as currently written. I'll mark this PR as a draft for now to avoid it accidentally being merged.